harness/determined/cli/trial.py

```diff
@@ -181,7 +181,17 @@ def download(args: Namespace) -> None:
             raise ValueError(f"No checkpoints found for trial {args.trial_id}")
         checkpoint = checkpoints[0]
 
-    path = checkpoint.download(path=args.output_dir)
+    try:
+        path = checkpoint.download(path=args.output_dir)
+    except errors.CheckpointStateException:
+        print(
+            f"Invalid Checkpoint State for checkpoint {checkpoint.uuid}.",
```

Comment:
print statement is weird. do we need newlines? you should be able to just do 
```
print(
        f"Invalid Checkpoint State for checkpoint {checkpoint.uuid}. "
        "It's checkpoint has likely been deleted. Please retry with a valid checkpoint UUID."
    )
```

Reply:
Changed it along with showing users checkpoint state.